src/App.js

```diff
@@ -1,29 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.